docs/admin/admin-ui-setup/dashboard-analysis.md

```diff
@@ -309,6 +309,16 @@ The dashboard has a graph of the Project over-quota GPU consumption.
 
 !![](img/consumption-dashboard-project-over-quota-graph.png)
 
+
+### Metrics used for consumption calculation
+
+* `runai_allocated_gpu_count_per_pod:hourly`&mdash;GPU allocation hour
```

Comment:
@lavianalon not clear on the deinition here. The metric is `hourly` but the description does not match. 
```suggestion
* `runai_allocated_gpu_count_per_pod:hourly`&mdash;GPU allocation hourly
```